sdks/java/core/src/main/java/org/apache/beam/sdk/io/DatastoreIO.java

```diff
@@ -387,6 +388,19 @@ public long getEstimatedSizeBytes(PipelineOptions options) throws Exception {
     }
 
     @Override
+    public void populateDisplayData(DisplayData.Builder builder) {
+      super.populateDisplayData(builder);
+      builder
+          .addIfNotDefault("host", host, DEFAULT_HOST)
+          .addIfNotNull("dataset", datasetId)
+          .addIfNotNull("namespace", namespace);
+
+      if (query != null) {
+        builder.add("query", query.toString());
```

Comment:
is `toString()` not the default implementation for the fallback method `addIfNotNull(String, Object)`?